src/math/Vector2.js

```diff
@@ -370,6 +370,18 @@ THREE.Vector2.prototype = {
 
 	},
 
+	angle: function () {
+
+		// computes the angle in radians with respect to the positive x-axis
+
+		var angle = Math.atan2( this.y, this.x );
+
+		if ( angle < 0 ) angle += 2 * Math.PI;
```

Comment:
I do not understand your comment. `atan2` returns a value between `- PI` and `+ PI`. A more natural range is zero to `2 PI`. That is what this method returns.